app/src/main/java/io/github/droidkaigi/confsched2017/repository/sessions/ContributorsRemoteDataSource.java

```diff
@@ -0,0 +1,25 @@
+package io.github.droidkaigi.confsched2017.repository.sessions;
```

Comment:
package should be `io.github.droidkaigi.confsched2017.repository.contributors;` not `io.github.droidkaigi.confsched2017.repository.sessions;`